packages/nimble-tokens/dist/fonts/css/fonts.css

```
@@ -6,6 +6,13 @@
   font-display: swap;
 }
 
+@font-face {
+  font-family: "Source Sans Pro-fallback";
```

preference, a name consistent with the current style, i.e. title case:
```suggestion
  font-family: "Source Sans Pro Fallback";
```

---

Changed.